core/pom.xml

```diff
@@ -95,6 +95,32 @@
         <artifactId>error_prone_annotations</artifactId>
         <version>2.13.1</version>
       </dependency>
+      <dependency>
+        <groupId>org.graalvm.nativeimage</groupId>
+        <artifactId>svm</artifactId>
+        <version>${graalvm.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.graalvm.sdk</groupId>
+        <artifactId>graal-sdk</artifactId>
+        <version>${graalvm.version}</version>
+      </dependency>
+      <dependency>
+        <!-- GAX provides native-image metadata for core Google libraries -->
+        <groupId>com.google.api</groupId>
+        <artifactId>gax</artifactId>
+        <version>2.16.0</version>
+        <exclusions>
+          <exclusion>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-api</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
```

Comment:
Should these dependencies be behind the profile for native testing?

Reply:
This is part of dependencyManagement. Having this here does not add new dependencies to library users. No effect to be here or in profile.

Reply:
I understand how dependencyManagement works. It's not true to say there is no effect - this dictates the version of these libraries for all dependencies, even if we don't necessarily need to dictate them. 

So the real question is "do these dependencies need to be included by default or should the behavior be limited to a profile". You're other comment seems to imply that it does, so we can follow up there.